.github/pull_request_template.md

```diff
@@ -37,6 +37,15 @@ In each section, please describe design decisions made, including:
 
 <!-- Some of the aspects mentioned above may be omitted for simple and small changes. -->
 
+#### Release note
+<!-- Give your best effort to summarize your changes in a couple of sentences aimed toward Druid users. 
+
+If your change doesn't have end user impact, you can skip this section.
+
+-->
+For tips about how to write a good release note, see [Release notes](../CONTRIBUTING.md#release-notes).
```

Comment:
the other links in the template are full links, I'm not certain if relative links work here or not, so to be safe we might want to make it match the others
```suggestion
For tips about how to write a good release note, see [Release notes](https://github.com/apache/druid/blob/master/CONTRIBUTING.md#release-notes).
```